Sources/PortalSwift/Core/Keychain/PortalKeychain.swift

```diff
@@ -247,7 +248,7 @@ public class PortalKeychain {
           ),
         ]
         return generateResponse
-      } catch KeychainError.itemNotFound(_) {
```

Comment:
Why doesn't this work anymore? I remember we tested this together too

Reply:
Yeah, I know. Not sure what happened, but it's not firing this error anymore 🤷